credsweeper/__main__.py

```diff
@@ -84,7 +84,7 @@ def get_arguments() -> Namespace:
                         help="parse .gitignore files and skip credentials from ignored objects",
                         dest="skip_ignored",
                         action="store_true")
-    parser.add_argument("--save-json",
+    parser.add_argument("--save_json",
```

Comment:
Let's change syntax in a different PR, as this change will break all scripts that using scanner CLI for sure :(
Alternatively, we might make both options supported, so both `save_json` and legacy `save-json` would work, [example](https://stackoverflow.com/a/11155124)

Reply:
Good point I'll make other PR for this.

Reply:
Revoked changes with [858ff7a](https://github.com/Samsung/CredSweeper/pull/125/commits/858ff7ac8d1c45bb635b61925fda0d9827a9e2e6).